src/CommunityToolkit.Maui.UnitTests/Extensions/ColorAnimationExtensions_Test.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using CommunityToolkit.Maui.Extensions;
+using CommunityToolkit.Maui.UnitTests.Mocks;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Extensions;
+
+public class ColorAnimationExtensions_Test : BaseTest
```

Comment:
Maybe add additional test where the `VisualElement` doens't have a `BackgroundColor` as I had to write logic to handle this scenario
```
[Fact]
public async Task BackgroundColorTo_VerifyColorChangedWhenInitialBackgroundNull()
{
	Color updatedBackgroundColor = Colors.Yellow;

	VisualElement element = MockAnimationHandler.Prepare(new Label());

	var isSuccessful = await element.BackgroundColorTo(updatedBackgroundColor);

	Assert.True(isSuccessful);
	Assert.Equal(updatedBackgroundColor, element.BackgroundColor);
}

```

Reply:
Great idea!

Done ✅